### DISCUSSION TITLE
Problems in parallel training on GPU

### DISCUSSION THREAD

**Discussion by qiurong13** [deepmodeling/deepmd-kit#1307] [Q&A]
With the help of Horovod,  the real batch size is 8 when batch_size is set to 2 in the input file and we launch 4 workers. According to the _Linear Scaling Rule_ in the attached paper _Accurate, Large Minibatch SGD: Training ImageNet in 1 Hour_: When the minibatch size is multiplied by k, multiply the learning rate by k.
Then how should we manually change the value of decay steps?
Why the decay steps should be reduced to 1/2 of steps in the above case?
In this relationship _lr(t) = start_lr * decay_rate ^ ( t / decay_steps )_, if t / decay_steps always equals to 200, then we could not possibly multiply the learning rate by k without changing decay_rate...

**Comment by wanghan-iapcm** [comment]
In the latest deepmd-kit, one does not need to set decay rate, but to set stop_lr instead.

**Reply by qiurong13** [reply]
Then how about the start_lr? do we have to set it manually? or Horovod will multiply it by k atomatically?

And, what is the rule to change decay steps? as in the manual, when batch size multiplies 4, decay steps decrease by half. but i think we can reduce it to 1/4 (1/k), as well as stop batch
